mycroft/util/parse.py

```diff
@@ -14,15 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-"""
-The mycroft.util.parse module provides various parsing functions for
-things like numbers, times, durations etc.
-
-The focus of these parsing functions is to extract data from natural speech
-and to allow localization.
-"""
-
```

Comment:
Why is this module level docstring removed?

Reply:
Looks like I didn't fully rebase somehow, thanks for catching it